1apply-css/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      Terri's Examples: Applying CSS to HTML With Cascade and Specificity
+    </title>
+    <link rel="stylesheet" href="1styles.css" />
+  </head>
+
+  <body>
+<div class="main-container">
```

Comment:
The class on this `div` is inconsistent with the block classes within. Generally speaking, the format should be `some-class-name` and then `some-class-name__block-name`. This will be important later on when you start to work in SCSS. e.g.
```scss
.main {
  color: red;
  &__paragraph {
    color: purple;
  }
}
```
-> transpiles to `.main__paragraph`.

Reply:
Thanks again for all the feedback. I went in and made the changes! :)